src/analysis/modals/CloudEnvironmentModal.ts

```diff
@@ -89,8 +89,8 @@ export const CloudEnvironmentModal = ({
   runtimes: Runtime[];
   apps: App[];
   appDataDisks: PersistentDisk[];
-  refreshRuntimes: () => Promise<void>;
-  refreshApps: () => Promise<void>;
+  refreshRuntimes: () => Promise<unknown>;
```

Comment:
I thought you found a workaround that enabled you to keep `Promise<void>`.

Reply:
missed one, apparently